Gemfile

```diff
@@ -1,27 +1,13 @@
 source "https://rubygems.org"
 ruby "2.7.4"
 
-plugin "bootboot", "~> 0.1.1"
```

Comment:
Removed bootboot (which helped dual boot our app in rails 5, rails 6.
Doing this since some of the changes we needed to make for rails 6 aren't backwards compatible, so we wouldn't be able to run with rails 5 anymore without conditionals distributed throughout the codebase

Reply:
Byeeeee